packages/gestalt/src/Link.js

```diff
@@ -81,20 +78,26 @@ type Props = {|
    */
   rounding?: Rounding,
   /**
-   * When `compress` is supplied, Link is visually compressed on click. See the [tapStyle and hoverStyle](https://gestalt.pinterest.systems/link#tapStyle-and-hoverStyle) variant for more information.
+   * When `compress` is supplied, Link is visually compressed on click.
    */
   tapStyle?: 'none' | 'compress',
   /**
    * Define the frame or window to open the anchor defined on href:
 - 'null' opens the anchor in the same window.
 - 'blank' opens the anchor in a new window.
 - 'self' opens an anchor in the same frame.
+
+See the [target variant](https://gestalt.pinterest.systems/link#Target) to learn more.
    */
   target?: null | 'self' | 'blank',
+  /**
+   * When `underline` is supplied, we override the underline style internally managed by the component. See the [underline variant](https://gestalt.pinterest.systems/link#Underline) to learn more.
+   */
+  underline?: 'none' | 'always' | 'hover',
```

Comment:
Given that this is an override for something we determine internally, would it make sense to provide an "auto" option? I'm thinking of a use case where the user would conditionally override the default underline style, and something like
```js
<Link underline={condition ? 'none' : 'auto'}>
```
communicates a bit better than
```js
<Link underline={condition ? 'none' : undefined}>
```
even though they're functionally the same.

(And "auto" feels more right than "default", since we're not specifying a default value — the default value is different according to the situation — but explicitly asking for the automatic selection behavior)

Reply:
actually I like that.

Reply:
DONE